### DISCUSSION TITLE
Delete previously optimized images.

### DISCUSSION THREAD

**Discussion by Valentim-Gab** [vercel/community#4328] [Help] [answered]
### Summary

I put up a study website on Vercel, but I didn't know it had image optimization and a limit of 1000. I've already disabled image optimization, but my limit of 1000 is still exceeded.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/85112333/ae8dbef1-8ade-4caa-8a48-a76d7a9a4476)
Preciso apagar essas imagens.

**Comment by amyegan** [comment, marked as answer]
Image Optimization usage is based on the number of unique [source images](https://vercel.com/docs/image-optimization/limits-and-pricing#source-images) requested during the billing period. The count of the source images is reset at the beginning of each billing period. This means that removal of images will not instantly lower your usage number because those resources have already been used.

These pages have more information about Image Optimization that can help you manage usage:
- https://vercel.com/docs/image-optimization/limits-and-pricing#hobby
- https://vercel.com/docs/image-optimization/managing-image-optimization-costs#minimizing-image-optimization-costs
- https://vercel.com/docs/image-optimization#caching
